pages/tissues/[uuid].js

```diff
@@ -191,15 +181,13 @@ export async function getServerSideProps({ params, req, query }) {
         tissue,
         documents,
         donors,
-        source,
         treatments,
-        biosampleTerm,
         diseaseTerms,
         pooledFrom,
         partOf,
         biomarkers,
         pageContext: {
-          title: `${tissue.biosample_term.term_name} — ${tissue.accession}`,
+          title: `${tissue.sample_terms[0].term_name} — ${tissue.accession}`,
```

Comment:
You can do it this way for the other pages too, like in-vitro system and whole organism. But let’s let that go for now to make sure this makes the release; it doesn’t hurt anything the way it is, but just not necessary.

Reply:
I applied this change to tissues, in vitro system, in whole organism, in technical sample and primary cell.